examples/release_highlights/plot_release_highlights_0_22_0.py

```diff
@@ -149,3 +149,39 @@
     # recomputed.
     estimator.set_params(isomap__n_neighbors=5)
     estimator.fit(X)
+
+############################################################################
+# Stacking Classifier (and Regressor)
+# -----------------------------------
+# These new estimators are a stack of estimators with a final classifier or
+# a regressor.
+# Stacked generalization consists in stacking the output of individual
+# estimator and use a classifier to compute the final prediction. Stacking
+# allows to use the strength of each individual estimator by using their output
+# as input of a final estimator.
+# Note that ``estimators_`` are fitted on the full ``X`` while
+# ``final_estimator``_ is trained using cross-validated predictions of the
+# base estimators using ``cross_val_predict`.
+
+from sklearn.datasets import load_iris
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.svm import LinearSVC
+from sklearn.linear_model import LogisticRegression
+from sklearn.preprocessing import StandardScaler
+from sklearn.pipeline import make_pipeline
+from sklearn.ensemble import StackingClassifier
+from sklearn.model_selection import train_test_split
+
+X, y = load_iris(return_X_y=True)
```

Comment:
For the docstring, it was enough but do you think that in the highlight we should use another dataset than `iris`?

Reply:
I think the highlights should be as easy as the docstrings, and the user guide is where the examples should be maybe more meaningful. We can link to the userguide for more info though.

Reply:
OK I see LGTM then